### PR TITLE
73 UI cleanup and flavor

### LINKED ISSUE #73: UI cleanup and flavor
As a Product Owner, I want to refresh the UI of our social-app-template to align more with current trends, specifically taking inspiration from modern social networks layout, to leverage the momentum from our "current ecosystem" and make our template more appealing and relevant.

Background:
Currently, our social-app-template has a layout similar to X (Twitter). While this layout is functional, there is an opportunity to capitalize on the current buzz surrounding new social networks and its their growth. We can make our template more engaging and relatable to the social networks of 2024.

Objectives:

- Redesign the social-app-template UI to resemble the moden social network web layout while keeping it generic enough to serve as a versatile example for users to interact with the Gateway.
- Ensure the new design aligns with the v1 functionality of Gateway and the social-app-template, maintaining clarity and ease of use.
- Leverage the momentum from our acquisition bid to make the template more appealing and relevant.

Key Considerations:

- Avoid a direct clone of any specific social network layout layout to maintain originality and relevance to our specific use cases.
- Focus on creating a modern and exciting user experience that resonates with users familiar with contemporary social networks.

Action Items:

- Schedule a meeting on Friday or Monday to scope out the redesign. @p5150j @claireolmstead (@wilwade optional)
- Define the elements of a modern social network layout that are most appealing and suitable for our template.
- Develop a plan to integrate these elements in a way that enhances the user experience without compromising the template's primary function as an example for Gateway users while keeping the template generic enough wherein users don't think this is a production application.

### PR DESCRIPTION
**Pull request by claireolmstead**
# Purpose
The goal of this PR is clean up the UI of the SAT frontend

Closes #73 

## Solution
Reformat and improve the UI of the SAT

@p5150j @mattheworris 

### Change summary
* Too many to list

## Steps to Verify
1. Run the app locally and poke around
2. If a bug is encountered, check that it's not already recorded in the SAT Projects Board.

## Additional details / screenshot
- 
Screenshot 2024-07-11 at 4 01 04 PM

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```
Comment (mattheworris) [inline .idea/.gitignore:5]: These `.idea` files snuck in again, why doesn't the root `.gitignore` keep these out?

File: frontend/src/content/FeedNav.tsx [hunk 2]
```diff
@@ -1,44 +1,68 @@
...
+import { useNavigate } from 'react-router-dom';
 
 interface FeedNavProps {
-  feedType: FeedTypes;
-  setFeedType: (type: FeedTypes) => void;
-  resetFeed: () => void;
-  goToMyFeed: () => void;
-  user?: User;
+  // feedType: FeedTypes;
+  // goToMyFeed: () => void;
```
Comment (mattheworris) [inline frontend/src/content/FeedNav.tsx:12]: nit: remove commented code
```suggestion
```

**Review by mattheworris — APPROVED** [review]
- Played around with the app, noted a few bugs in edge cases and will create issues for them
- There might be a compatibility issue with `content-publishing-service:0.11.0`, still tracking it down, tested with `0.10.1`
- Read through the code quickly
🚢 it!

File: .idea/.gitignore (hunk 1 above)
Comment (claireolmstead) [reply .idea/.gitignore:5]: IDK. Annoying.

**Pull request merged**